package.json

```diff
@@ -50,9 +51,22 @@
     "elasticsearch": "^3.1.1",
     "express": "^4.10.6",
     "glob": "^4.3.2",
+    "good": "^5.1.2",
+    "good-console": "^4.1.0",
+    "good-file": "^4.0.2",
+    "good-reporter": "^3.1.0",
+    "hapi": "^8.6.1",
     "http-auth": "^2.2.5",
+    "jade": "^1.8.2",
```

Comment:
Some of these packages should probably be removed https://gist.github.com/anonymous/fb71afb1e214d2605f3b